api.go

```diff
@@ -117,9 +121,15 @@ type (
 		// - P240p30fps16x9
 		// - P240p30fps4x3
 		// - P144p30fps16x9
-		Profiles            []Profile `json:"profiles,omitempty"`
-		Record              bool      `json:"record,omitempty"`
-		RecordObjectStoreId string    `json:"recordObjectStoreId,omitempty"`
+		Profiles            []Profile      `json:"profiles,omitempty"`
+		Record              bool           `json:"record,omitempty"`
+		RecordObjectStoreId string         `json:"recordObjectStoreId,omitempty"`
+		PlaybackPolicy      PlaybackPolicy `json:"playbackPolicy,omitempty"`
```

Comment:
Nit could omit the field name since same as type

```suggestion
		*PlaybackPolicy `json:"playbackPolicy,omitempty"`
```

Also are you sure omitempty will work for structs? Might be better to make it a pointer instead with a clear definition of empty: nil

Reply:
Pretty sure it works, I think I did the same in another occasion, will test it tho. I am importing this hash commit into the record tester to use it while developing